pipeline/pre_processer.py

```diff
@@ -19,10 +21,46 @@ def pre_processer(args):
     np.savez(meta.path_to_binning, bin_low=bin_low, bin_high=bin_high,
              bin_center=bin_center)
 
-    # Second step is to create the survey mask from a hitmap
+    if args.plots:
+        plt.figure(figsize=(8, 6))
+        for i in range(len(bin_low)):
+            plt.fill_between([bin_low[i], bin_high[i]], [0, 0], [i+1, i+1],
+                             alpha=0.5)
+        plt.xlabel(r'$\ell$', fontsize=14)
+        plt.ylabel('bin index', fontsize=14)
+        plt.title('Binning', fontsize=14)
+        plt.savefig(meta.path_to_binning.replace('.npz', '.png'))
+
+    # Second step is to download hits map and apodized analysis mask
+    print("Download and save SAT hits map ...")
```

Comment:
maybe we can remove the following block which downloads the hitmap since it's now done in mask_handler.py

Reply:
Done!